framework/filters/AccessRule.php

```diff
@@ -90,6 +92,47 @@ class AccessRule extends Component
      */
     public $denyCallback;
 
+    /** 
+     * @var array a map for passing parameters to the [[User::can()]] function
+     * 
+     * The map indicates query parameters that will be passed when determining
+     * role access.
+     * 
+     * If only a value is passed then the query parameter will by passed as is.
```

Comment:
Basically it means that passing only a value is equivalent to passing `value => value`. Not sure how to phrase it better